### ISSUE TITLE
[python bindings] can't import eigenpy and Quaternion within the same script

### ISSUE THREAD

**Issue by stonneau** [stack-of-tasks/pinocchio#816]
Hi, with the latest version of Pinocchio, the following two lines won't work:

```python
import eigenpy;
from pinocchio import Quaternion;
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-cac19b39cca1> in <module>()
----> 1 from pinocchio import Quaternion

ImportError: cannot import name Quaternion
```

Conversely, the other way around things seem fine. This is not high priority.

Thanks 

Steve

**Comment by nim65s** [comment]
Quaternion is in eigenpy, isn't it ?

```
In [1]: import eigenpy

In [2]: eigenpy.Quaternion
Out[2]: eigenpy.Quaternion
```

**Comment by stonneau** [comment]
It is, however the import is not mine, but comes from a file that I include.
If your answer is the answer to the issue, then I guess it means Quaternion must be removed from pinocchio don't you think ?

**Comment by nim65s** [comment]
No, sorry, I misunderstood the question. Yes this is an issue, and yes this is minor :)

**Comment by jcarpent** [comment]
I think the bug is related to EigenPy instead. It is due to the default exposition of Quaternion by the two applications. So if eigenpy is loaded first, then pinocchio will not expose it and the class Quaternion won't appear in the namespace of Pinocchio.

Thanks @stonneau for raising this bug.

**Comment by jcarpent** [comment]
Solved by #817
